app/src/main/java/com/kickstarter/ui/fragments/PledgeFragment.kt

```diff
@@ -21,19 +21,23 @@ import android.widget.TextView
 import androidx.annotation.NonNull
 import androidx.constraintlayout.widget.ConstraintSet
 import androidx.core.content.ContextCompat
+import androidx.core.view.isGone
```

Comment:
Tested 
- Pledge adding a new card in the pledge : ok
- Pledge using and existing card : ok
- Pledge with add-ons with shipping: ok
- Update pledge without add ons: ok
- Cancel pledge: ok
- Change payment method without addons: ok
- Change reward: ok